circuit_maintenance_parser/parsers/turkcell.py

```diff
@@ -0,0 +1,68 @@
+"""Turkcell parser."""
+import logging
+import re
+from typing import Dict
+
+from bs4.element import ResultSet  # type: ignore
+from dateutil import parser
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-nested-blocks, too-many-branches
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserTurkcell1(Html):
+    """Notifications Parser for Turkcell notifications."""
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_tables(soup.find_all("table"), data)
+            return [data]
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_tables(self, tables: ResultSet, data: Dict):
+        """Parse tables."""
+        # Main table
+        td_elements = tables[0].find_all("td")
+        for idx, td_element in enumerate(td_elements):
+            if "Dear Customer" in td_element.text.strip():
+                if "planned" in td_element.text.strip():
+                    data["status"] = Status["CONFIRMED"]
+                else:
+                    data["status"] = Status["CONFIRMED"]
+            if "Maintenance Number" in td_element.text.strip():
+                data["maintenance_id"] = td_elements[idx + 1].text.strip()
+            elif "Start" in td_element.text.strip():
+                data["start"] = self.dt2ts(parser.parse(td_elements[idx + 1].text.strip()))
+            elif "End" in td_element.text.strip():
+                data["end"] = self.dt2ts(parser.parse(td_elements[idx + 1].text.strip()))
+            elif "Impact of the maintenance" in td_element.text.strip():
+                data["summary"] = td_elements[idx + 1].span.text.strip()
+                if len(tables) == 1:
```

Comment:
if I understand correctly, the Circuit info can be part of the "Impact of the maintenance" section or in a separate table?
maybe, to make this more evident, in this "elif" you could get the `p_elements` and then implement the if statement at the end:

```python
if len(tables) == 1 and p_elements:
...
elif len(tables) == 2:
...

```